src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/feature/NoviflowSpecificFeature.java

```diff
@@ -51,7 +64,7 @@ boolean is100GbHw(IOFSwitch sw) {
                 .orElse("")).matches();
     }
 
-    boolean isVirtual(IOFSwitch sw) {
+    static boolean isVirtual(IOFSwitch sw) {
```

Comment:
let's rename it to something like isSMSeries

Reply:
fixed